api/desecapi/tests/test_domains.py

```diff
@@ -109,6 +109,7 @@ def test_cant_register_internal(self):
         self.assertNotRegistrable('internal')
         self.assertNotRegistrable('catalog.internal')
         self.assertNotRegistrable('some.other.internal')
+        self.assertNotRegistrable('ending.in.a.dot.internal')
```

Comment:
the trailing dot's missing :)

Reply:
`Domain.is_registrable()` assumes there is no trailing dot at the end. This is because the function is called from within `DomainSerializer.validate_name()` which runs only after the name regex has been applied. Outside of tests, it cannot happen that the argument of `Domain.is_registrable()` has a dot at the end.

If it happens in a test, `Domain.is_registrable()` throws an uncaught exception. We may or may not want to fix this, but in any case it has nothing to do with the commits in this PR. If you think it's worth fixing, let's open an issue.